### ISSUE TITLE
Improvement: There should be focused state for interactive components

### ISSUE THREAD

**Issue by NEWESTERS** [ardakaracizmeli/design-system-checklist#36]
For interactive components such as:

- [ ] Button
- [ ] Dropdown
- [ ] Input Checkbox
- [ ] Input Radio
- [ ] Input Text
- [ ] Input Switch
- [ ] Tabs

there should be focused state for keyboard accessibility

**Comment by ardakaracizmeli** [comment]
Hello, this already exists, both in guidelines, and in our code.
